tailcfg/tailcfg.go

```diff
@@ -149,7 +149,8 @@ type CapabilityVersion int
 //   - 104: 2024-08-03: SelfNodeV6MasqAddrForThisPeer now works
 //   - 105: 2024-08-05: Fixed SSH behavior on systems that use busybox (issue #12849)
 //   - 106: 2024-09-03: fix panic regression from cryptokey routing change (65fe0ba7b5)
-const CurrentCapabilityVersion CapabilityVersion = 106
+//   - 107: 2024-10-30: add App Connector to conffile (PR #13942)
+const CurrentCapabilityVersion CapabilityVersion = 107
```

Comment:
I assume that this is the only place where I need to change this? (apart from the changes in operator/proxy config PR)

Reply:
Technically this doesn't even need a capver bump? The control server doesn't care, does it?

But fine if you want. I'd drop the space in AppConnector to match the symbol if you keep it

Reply:
> Technically this doesn't even need a capver bump?

We use the capver to determine which clients (proxies) can read what conffiles (because [the deserialization fails on unknown fields as is now](https://github.com/tailscale/tailscale/blob/1103044598ac2897a3f2f6687dc9d2b3d23f7da5/ipn/conffile/conffile.go#L91) and we want to support older proxies/newer operators)

Reply:
Going to merge this today if there are no further comments

Reply:
Merge away.

If `type AppConnectorPrefs struct` ever drifts in a way we don't want in the conf file, we can always fork the type later.